.changeset/mean-tigers-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+Allow asides titles to contain markdown formatting
```

Comment:
Small suggestion for the changeset base on our [style guide](https://contribute.docs.astro.build/docs-for-code-changes/pull-requests/#changesets)

```suggestion
Adds support for markdown formatting in aside titles
```